### PR TITLE
BUG: correctly translate # of days into # of minutes for simple transforms

### PR DESCRIPTION
**Pull request by jbredeche**

### CODE REVIEW

File: zipline/data/data_portal.py [hunk 1]
```diff
@@ -1739,6 +1740,31 @@ def get_fetcher_assets(self, dt):
         else:
             return [assets] if isinstance(assets, Asset) else []
 
+    @lru_cache(8)
```
Comment (llllllllll) [inline zipline/data/data_portal.py]: why 8 exactly?

File: zipline/data/data_portal.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 from operator import mul
 
 import bcolz
+from cachetools import lru_cache
```
Comment (llllllllll) [inline zipline/data/data_portal.py]: this is available in functools32, if we add this we should also update `memoize_last` in zipline.utils.memoize to be implemented as `lru_cache(1)`

File: zipline/data/data_portal.py [hunk 3]
```diff
@@ -1739,6 +1740,31 @@ def get_fetcher_assets(self, dt):
...
+        )
+
+        minutes_count = \
+            sum(210 if day in self.env.early_closes else 390 for day in days)
+
+        # add the minutes for today
+        today_open = self.env.get_open_and_close(ending_minute)[0]
+
+        minutes_count += int(
+            (ending_minute - today_open).total_seconds() / 60
```
Comment (llllllllll) [inline zipline/data/data_portal.py]: instead of `int(a / b)` we can use `a // b`

File: zipline/data/data_portal.py (hunk 2 above)
Comment (jbredeche) [inline zipline/data/data_portal.py]: good call, switched and done.

**Comment by jbredeche** [comment]
thanks @llllllllll for the py2/py3 compatibility work!

File: setup.py [hunk 4]
```diff
@@ -189,15 +198,22 @@ def read_requirements(path, strict_bounds, conda_format=False):
...
 
         return list(reqs)
 
 
 def install_requires(strict_bounds=False, conda_format=False):
-    return read_requirements('etc/requirements.txt',
+    reqs = read_requirements('etc/requirements.txt',
                              strict_bounds=strict_bounds,
                              conda_format=conda_format)
+    if sys.version_info.major == 2 or conda_format:
```
Comment (richafrank) [inline setup.py]: Why do we need the `or conda_format` portion?

Comment (llllllllll) [inline setup.py]: I am pretty sure this is executed with the python from the conda root

**Pull request merged**